core/src/test/scala/unit/kafka/utils/CommandLineUtilsTest.scala

```diff
@@ -26,30 +26,53 @@ class CommandLineUtilsTest {
   @Test(expected = classOf[java.lang.IllegalArgumentException])
   def testParseEmptyArg() {
     val argArray = Array("my.empty.property=")
-    CommandLineUtils.parseKeyValueArgs(argArray, false)
+    val acceptMissingValue = false;
+
+    CommandLineUtils.parseKeyValueArgs(argArray, acceptMissingValue)
```

Comment:
@mutdmour In scala, you can write this as:
```
CommandLineUtils.parseKeyValueArgs(argArray, acceptMissingValue = false)
```
and
```
CommandLineUtils.parseKeyValueArgs(argArray, acceptMissingValue = true)
```
That makes the code more readable. Same for the other uses of `CommandLineUtils.parseKeyValueArgs` in this file with the boolean.

Reply:
oh I see. that's clearer. will fix it up.